src/SqlStreamStore.Postgres/PgSqlScripts/MigrationOld.sql

```diff
@@ -0,0 +1,74 @@
+COMMENT ON SCHEMA __schema__ IS '{ "version": 3 }';
+
+DROP FUNCTION public.read_all(int4, int8, bool, bool);
```

Comment:
This would remove an in active function in prod with our idea.
Wouldn't do this